### PR TITLE
Add optional output log file to server CLI

### LINKED ISSUE #23: [Logger] Define output
Currently the logger is hardcode to the STDOUT, which may be not a very good option for every user, what about defining this in the CLI?

e.g.: `bojack server --log ./shared/logs/bojack.log` 

BoJack would have to create a new file, with the given prefix and add a timestamp:

e.g.:  `./shared/logs/bojack_201609201010.log`

### PR DESCRIPTION
**Pull request by hugoabonizio**
Closes #23.

### CODE REVIEW

File: src/bojack/cli.cr [hunk 1]
```diff
@@ -42,13 +40,30 @@ cli = Commander::Command.new do |command|
...
+    command.flags.add do |flag|
       flag.name = "log-level"
       flag.long = "--log-level"
       flag.default = 1
       flag.description = "Log severity."
     end
 
     command.run do |options, arguments|
+      output = if options.string["log"].empty?
+                 STDOUT
```
Comment (mauricioabreu) [inline src/bojack/cli.cr:58]: This indentation seems weird o.o
Or am I crazy?

**Comment by mauricioabreu** [comment]
Cool job 👍

File: src/bojack/cli.cr (hunk 1 above)
Comment (hugoabonizio) [reply src/bojack/cli.cr:58]: I agree with you, but that's the Crystal standard, I'm using https://github.com/crystal-lang/sublime-crystal

Comment (marceloboeira) [reply src/bojack/cli.cr:58]: that is the if for assignment indentation, ruby-like.

Comment (hugoabonizio) [reply src/bojack/cli.cr:58]: I think it's very ugly but very useful at the same time hahah

Comment (marceloboeira) [reply src/bojack/cli.cr:58]: @hugoabonizio exactly  ahaha

**Pull request merged**